DXMainClient/DXGUI/Multiplayer/GameInformationPanel.cs

```diff
@@ -156,15 +201,54 @@ public void ClearInfo()
             lblHost.Visible = false;
             lblPing.Visible = false;
             lblPlayers.Visible = false;
+            lblGameInformation.Visible = false;
 
             foreach (XNALabel label in lblPlayerNames)
                 label.Visible = false;
+
+            if (mapTexture != null && !mapTexture.IsDisposed)
+                mapTexture.Dispose();
```

Comment:
I am not sure but I suspect this can be done as just `mapTexture?.Dispose()`, also should probably null `mapTexture` after?

Reply:
Tested, and that worked